src/WebdriverClassicDriver.php

```diff
@@ -1,21 +1,1345 @@
 <?php
 
-/*
+/**
  * This file is part of the Behat\Mink.
  * (c) Konstantin Kudryashov <ever.zet@gmail.com>
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
+ *
+ * @noinspection PhpLanguageLevelInspection
  */
 
 namespace Mink\WebdriverClassDriver;
 
 use Behat\Mink\Driver\CoreDriver;
+use Behat\Mink\Exception\DriverException;
+use Behat\Mink\Selector\Xpath\Escaper;
+use Facebook\WebDriver\Exception\NoSuchCookieException;
+use Facebook\WebDriver\Exception\NoSuchElementException;
+use Facebook\WebDriver\Exception\WebDriverException;
+use Facebook\WebDriver\Remote\DesiredCapabilities;
+use Facebook\WebDriver\Remote\RemoteWebDriver;
+use Facebook\WebDriver\Remote\RemoteWebElement;
+use Facebook\WebDriver\WebDriverBy;
+use Facebook\WebDriver\WebDriverDimension;
+use JetBrains\PhpStorm\Language;
+use JsonException;
+use Throwable;
 
 class WebdriverClassicDriver extends CoreDriver
 {
+    public const DEFAULT_BROWSER = 'chrome';
+
+    public const DEFAULT_CAPABILITIES = [
+        'default' => [
+            'platform' => 'ANY',
+            'name' => 'Behat Test',
+            'deviceOrientation' => 'landscape',
+            'deviceType' => 'desktop',
+        ],
+        'chrome' => [
+            'goog:chromeOptions' => [
+                'excludeSwitches' => ['enable-automation'],
+            ],
+        ],
+        'firefox' => [
+        ],
+    ];
+
+    private const W3C_WINDOW_HANDLE_PREFIX = 'w3cwh:';
+
+    private ?RemoteWebDriver $webDriver = null;
+
+    private string $browserName;
+
+    private DesiredCapabilities $desiredCapabilities;
+
+    private bool $started = false;
+
+    private array $timeouts = [];
+
+    private Escaper $xpathEscaper;
+
+    private string $webDriverHost;
+
+    private ?string $initialWindowName = null;
+
+    /**
+     * @throws DriverException
+     */
+    public function __construct(
+        string  $browserName = null,
+        array   $desiredCapabilities = null,
```

Comment:
I don't like it either, to be honest.

Reply:
We generally even keep the arguments on a single line.

Reply:
Ever since arguments (and especially, constructors' arguments) in PHP got fancier (typing, attributes and so on), I found it easier to have them on separate lines.
Additionally, it also improves readability of git 'blame' history.